common/app/model/content.scala

```diff
@@ -451,7 +451,8 @@ object Article {
       javascriptConfigOverrides = javascriptConfig,
       opengraphPropertiesOverrides = opengraphProperties,
       shouldHideHeaderAndTopAds = (content.tags.isTheMinuteArticle || (content.isImmersive && (content.elements.hasMainMedia || content.fields.main.nonEmpty))) && content.tags.isArticle,
-      contentWithSlimHeader = (content.isImmersive && content.tags.isArticle) || content.showNewRecipeDesign
+      contentWithSlimHeader = (content.isImmersive && content.tags.isArticle),
+      isNewRecipe = content.showNewRecipeDesign
```

Comment:
I'm indifferent. I feel both shouldn't be there, but for consistency reasons you are absolutely right.